contrib/time_series_expectations/time_series_expectations/expectations/column_aggregate_time_series_expectation.py

```diff
@@ -0,0 +1,102 @@
+from abc import ABC
+from typing import Dict, Optional
+
+import pandas as pd
+
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+from great_expectations.execution_engine import (
+    ExecutionEngine,
+)
+from great_expectations.expectations.expectation import ColumnExpectation
+from time_series_expectations.expectations.util import get_prophet_model_from_json
+
+
+class ColumnAggregateTimeSeriesExpectation(ColumnExpectation, ABC):
+    """This Expectation is used to compare the values in a column to a Prophet forecast, based on a timestamp in a second column."""
+
+    # These examples will be shown in the public gallery.
+    # They will also be executed as unit tests for your Expectation.
+    # examples = []
+
+    # metric_dependency
+
+    # Expectations of this type can only have a single metric dependency.
+    @property
+    def metric_dependencies(self):
+        return (self.metric_dependency,)
+
+    # This a tuple of parameter names that can affect whether the Expectation evaluates to True or False.
+    # success_keys = ("min_value", "strict_min", "max_value", "strict_max")
+    success_keys = ("model", "date", "column")
```

Comment:
Could we add more to the docstring to clarify what is acceptable in these arguments (incl type)? Using an `Args:` section like in the google python style guide.

Reply:
Yep. Added a better docstring.